duration duration
float32 emphasis
---
bool response
